packages/golden-layout/tsconfig.json

```diff
@@ -5,5 +5,6 @@
     "outDir": "dist/"
   },
   "include": ["src/**/*.ts", "src/**/*.tsx", "src/**/*.js", "src/**/*.jsx"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules"],
+  "references": [{ "path": "../components" }]
```

Comment:
Good catch. Although it's only used for scss so probably why it never caused an issue

Reply:
True